src/components/textarea/mock-handlers.js

```diff
@@ -0,0 +1,18 @@
+import { rest } from 'msw';
+import mockConstants from '../../mocks/mock-constants.js';
```

Comment:
non-blocking question: why do we need to do anything with mock-handlers for a textarea component?

Reply:
I wouldn't say we _need_ to, more that this provided a nice way to do a test around an async update of the `value` prop to catch that class of bugs.